packages/realtime_client/test/mock_test.dart

```diff
@@ -38,41 +38,6 @@ void main() {
                   ['postgres_changes']
               .first['filter'];
 
-          final replyString = jsonEncode({
```

Comment:
Whats the reason for this code block to be removed?

Reply:
Yeah, on line 72 here we have `'topic': 'realtime:public:todos'`, and on each of the test cases, we have `client.channel('public:todoos')`.

On this reply, we were replying to `todos`, but the channel each of the test client was subscribed to was `todoos` (extra "o"), which means this reply was ignored in the tests, and never was doing anything. 

I think we copied it over from the tests on supabase package [here](https://github.com/supabase/supabase-flutter/blob/main/packages/supabase/test/mock_test.dart#L132) and didn't notice the typo on the channels until now, therefore just had some dead code.